docs/code-quality/ca2241.md

```diff
@@ -10,7 +10,6 @@ helpviewer_keywords:
 - ProvideCorrectArgumentsToFormattingMethods
 - CA2241
 ms.assetid: 83639bc4-4c91-4a07-a40e-dc5e49a84494
-author: mikejo5000
```

Comment:
Why was this deleted? Should this change be reverted?

Reply:
Yes, not sure why that was deleted.

Reply:
Added